classWork/PhotoEnhancer/PhotoEnhancer/Filters/Transform/LeftTransformer.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PhotoEnhancer
+{
+    public class LeftTransformer : ITransformer<LeftParameters>
```

Comment:
Название класса неудачное. Из него непонятно, что будет происходить с изображением. Лучше так
```suggestion
    public class LeftSkewTransformer : ITransformer<LeftParameters>
```